conans/client/conan_api.py

```diff
@@ -386,7 +395,7 @@ def download(self, reference, remote=None, package=None, recipe=False):
             raise ConanException("recipe parameter cannot be used together with package")
         # Install packages without settings (fixed ids or all)
         conan_ref = ConanFileReference.loads(reference)
-        self._manager.download(conan_ref, package, remote=remote, recipe=recipe)
+        self._manager.download(conan_ref, package, remote=remote, only_recipe=recipe)
```

Comment:
rename not related with this PR, it makes more sense for me to call it "only_recipe".